kintone/js-sdk
# refactor(plugin-packer): Migrate to TypeScript

## Why



To improve maintainability.

## What



Migrate to TypeScript. Just a refactoring. This PR is not strictly typed.

## How to test



- Run the same tests as before.
- Check behaviors on site preview manually (https://pr-671.d1ite8zvzg8c5i.amplifyapp.com/).

## Todo

- [x] Rebase and resolve conflicts for using #670
- [x] Migrate JavaScript files under `./test` to TypeScript

## Checklist
